sbncode/TPCReco/VertexStub/StubMergeAlgorithms.cxx

```diff
@@ -49,14 +52,17 @@ geo::Point_t sbn::GetLocationAtWires(const spacecharge::SpaceCharge *sce, geo::P
 double sbn::GetPitch(
     const geo::WireReadoutGeom &wireReadout, const spacecharge::SpaceCharge *sce,
     geo::Point_t loc, geo::Vector_t dir, 
-    geo::View_t view, geo::TPCGeo const& tpc,
+    geo::View_t view, geo::PlaneID const& plane_id,
     bool correct_sce, bool track_is_sce_corrected, float xsign) {
 
-  double angleToVert = wireReadout.WireAngleToVertical(view, tpc.ID()) - 0.5*::util::pi<>();
+  const geo::GeometryCore *geo = lar::providerFrom<geo::Geometry>();
+
+  double angleToVert = wireReadout.WireAngleToVertical(view, plane_id) - 0.5*::util::pi<>();
```

Comment:
By the way, this is the direction from `geo->Plane(plane_id).GetIncreasingWireDirection()`, which would make `cosgamma` a simple `std::abs(geo::vect::Dot(dirToVert, dir_w))`.